adapter/src/main/resources/Dockerfile

```diff
@@ -56,4 +56,7 @@ COPY maven/adapter-linux-amd64 adapter
 COPY maven/check_health.sh .
 COPY maven/LICENSE.txt .
 
+# DEBUG_LEVEL is for debugging go-amqp library.
+ENV DEBUG_LEVEL=0
```

Comment:
Shall we add the same to the ubuntu image as well?

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2658/commits/a82fced9ee0645ba7522cacf42b3b365b5e89122